src/components/Prose.astro

```diff
@@ -1,13 +1,16 @@
+---
+/* eslint-disable tailwindcss/no-custom-classname */
```

Comment:
The prose-headings class wasn't recognized even though it's valid. This will be because it's through the tailwind-typography plugin and not a normal Tailwind class.